components/tutorials/TutorialsPane.tsx

```diff
@@ -7,8 +7,20 @@ import classNames from 'classnames'
 
 export function TutorialsPane({ className }: SectionProps) {
   return (
-    <div className="grid grid-cols-2 gap-x-6">
-      <div className="col-span-2">
+    <div
+      className="
+      container grid grid-cols-2 gap-x-6
+      tablet:grid-cols-1 tablet:grid-x-0 tablet:w-full tablet:grid-gap-x-0
```

Comment:
I think this format makes the code harder to read - let's keep what Nicolas used in the first place